tool/loadgen/save.go

```diff
@@ -15,8 +15,11 @@ import (
 type TestResult struct {
 	FirstAppResponse string
 	*vegeta.Metrics
-	Stats        map[string]Stats       `json:"container_stats"`
-	RelayMetrics map[string]interface{} `json:"relay_metrics,omitempty"`
+	Stats          map[string]Stats       `json:"container_stats"`
+	RelayMetrics   map[string]interface{} `json:"relay_metrics,omitempty"`
+	LoadGenOptions struct {
+		Command []string
```

Comment:
Did you envision adding more fields here?

If not, this could be flattened to get rid of the struct or even the slice as well, since when we render the template we're throwing away the individual args and joining everything together.

Reply:
I left it purposely broad, but considering the potential delta of changes is pretty small to add new option, I'll just push a change to flatten the command.